update-bcd.ts

```diff
@@ -414,7 +425,7 @@ export const update = (
             simpleStatement.version_added !== 'preview' &&
             compareVersions(
               version,
-              simpleStatement.version_added.replace('≤', ''),
+              splitRange(simpleStatement.version_added).upper,
```

Comment:
Is it really possible for `simpleStatement.version_added` to contain a `${lower}> ≤${upper}` range? Is this to handle the case where the update script is run twice?

This wasn't supported previously, and there aren't any tests that have this format in the initial data. If you want to support this, can you add a test?

Reply:
I don't want to introduce support for that condition, no. I thought using the utility function here (rather than a call to `replace`) would better communicate the code's intent. That's debatable, but it definitely makes the code unnecessarily forgiving. I've removed that application and replaced the code path designed to support it with a `throw` statement. Much safer!

Reply:
Great, thanks!